packages/patternfly-4/react-charts/src/components/ChartArea/examples/ChartArea.md

```diff
@@ -26,13 +25,13 @@ Learn to build an area chart using a Katacoda tutorial starting with a simple ch
 [Start course](https://katacoda.com/patternfly/courses/charts/area-chart)
 
 ## Examples
-```js title=Right-aligned-legend
+```js title=Right--aligned-legend
```

Comment:
The title of the 1st example should be "Simple area chart with right aligned legend"